tracer/src/Datadog.Trace/Iast/IastModule.cs

```diff
@@ -904,6 +904,10 @@ public void UnregisterDbRecord(object instance)
             dataBaseRows.Remove(instance);
         }
 
+        // note: the return value this method indicates if the value could have been tainted
+        // if a iast context is available. It will return true if iast is enabled and the row
+        // count conditions are met. The result is used in the unit tests, don't use it
+        // anywhere else
```

Comment:
nit: should we add this comment to the doc comments? 🙂 
```suggestion
        /// <summary>
        /// Add a DB value to be tainted
        /// </summary>
        /// <returns>The return value this method indicates if the value could have been tainted
        /// if a iast context is available. It will return true if iast is enabled and the row
        /// count conditions are met. The result is used in the unit tests, don't use it
        /// anywhere else
        /// </returns>
```

Reply:
fixed in c59e762ba